commons/src/main/java/io/hyscale/commons/utils/HyscaleStringUtil.java

```diff
@@ -45,4 +46,27 @@ public static String removeSuffixChar(StringBuilder input, char trailingChar) {
         return removeSuffixStr(input.toString(), Character.toString(trailingChar));
     }
 
+
+    public static String getPlural(String word) {
```

Comment:
Can you check if we can use a library, instead of maintaining this code at our end?

Reply:
Will be taking this for next release, as it is less priority.